<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KoreaST8ChGUIForm</class>
 <widget class="GUIFormBase" name="KoreaST8ChGUIForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>85</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDial" name="dialLightValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLightValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
